src/odin/enhancedtrippath.cc

```diff
@@ -918,6 +930,9 @@ std::string EnhancedTripLeg_Edge::TurnLanesToString() const {
     if (turn_lane.is_active()) {
       str += " ACTIVE";
     }
+    if (turn_lane.is_possible()) {
```

Comment:
Can this be `else if` so that we don't mark all active lanes as possible in RAD output? Since active implies possible as well.

Reply:
I had originally kept these 2 attributes mutually exclusive when setting the values in the protobuf. But yeah, you make a good point that RAD will get affected by this change (I haven't completed RAD testing yet, so didn't realize this will be an issue). Technically it will make the string representation of lanes not match the corresponding values in the protobuf, but I guess thats better than changing many tests & will make RAD easier.